output/components.eslint.txt

```diff
@@ -17,6 +17,13 @@ The react/require-extension rule is deprecated. Please use the import/extensions
 /home/travis/build/Talend/ui/packages/components/src/HeaderBar/HeaderBar.component.js
   105:37  error  Elements with ARIA roles must use a valid, non-abstract ARIA role  jsx-a11y/aria-role
 
+/home/travis/build/Talend/ui/packages/components/src/Notification/Notification.component.js
+   9:27  error  Absolute imports should come before relative imports                                 import/imports-first
```

Comment:
Most of those errors seem to be easily fixed, can you have a look at it ?

Reply:
Thanks, fixed